gorapass/models.py

```diff
@@ -17,7 +17,8 @@ def __str__(self):
       return self.stamp_name
 
 class Hikes(models.Model):
-    stamp_name = models.TextField()   #.ForeignKey(Stamps, to_field='stamp_name', on_delete=models.CASCADE)
+    stamp = models.ForeignKey(Stamps, on_delete=models.CASCADE)
+    stamp_name = models.TextField() ##.ForeignKey(Stamps, to_field='stamp_name', on_delete=models.CASCADE)
```

Comment:
What are these comments about? We're still keeping a 'stamp_name' column on our `Hikes` model?

Reply:
I will remove the comments - I forgot to delete them before

Reply:
I will remove stamp_name from the model